src/timers.cc

```diff
@@ -122,6 +122,10 @@ double ProcessCPUUsage() {
   // same as total time, but this is ok because there aren't long-latency
   // syncronous system calls in Emscripten.
   return emscripten_get_now() * 1e-3;
+#elif defined(BENCHMARK_OS_SOLARIS)
+  struct rusage ru;
```

Comment:
Isn't this identical to the `#else` case? Is that intended?

Reply:
Yes, it is, thank you for catching it. Removed.